__tests__/selenium_tests/e2e_tests.ts

```diff
@@ -90,7 +90,10 @@ describe('End to end test', () => {
         logger.info('waiting until url contains model')
         await driver.wait(until.urlContains('/model/'))
         const modelUrl = await driver.getCurrentUrl()
-        const mName = modelUrl.match('/.*/model/(?<name>[^/]*)')!.groups!.name
+
+        const match = modelUrl.match('/.*/model/(?<name>[^/]*)')
+        if (!match || !match.groups) throw new Error('Could not parse model UUID from URL')
```

Comment:
This will have conflicts with my branch that has finally been merged in. I prefer your solution so feel free to override my changes